android/java/org/chromium/chrome/browser/vpn/split_tunnel/ApplicationViewModel.java

```diff
@@ -0,0 +1,83 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.chromium.chrome.browser.vpn.split_tunnel;
+
+import android.Manifest;
+import android.app.Activity;
+import android.content.pm.ApplicationInfo;
+import android.content.pm.PackageInfo;
+import android.content.pm.PackageManager;
+
+import androidx.lifecycle.MutableLiveData;
+import androidx.lifecycle.ViewModel;
+
+import org.chromium.chrome.browser.vpn.utils.BraveVpnPrefUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+public class ApplicationViewModel extends ViewModel {
+    MutableLiveData<List<ApplicationDataModel>> excludedApplicationDataLiveData;
+    MutableLiveData<List<ApplicationDataModel>> applicationDataLiveData;
+    MutableLiveData<List<ApplicationDataModel>> systemApplicationDataLiveData;
+
+    public ApplicationViewModel() {
+        excludedApplicationDataLiveData = new MutableLiveData<>();
+        applicationDataLiveData = new MutableLiveData<>();
+        systemApplicationDataLiveData = new MutableLiveData<>();
+    }
+
+    public MutableLiveData<List<ApplicationDataModel>> getExcludedApplicationDataLiveData() {
```

Comment:
I suppose only `LiveData` should be exposed to other classes. Or is it done due to some specific reason?